src/components/Accordion/Accordion/Accordion.jsx

```diff
@@ -11,7 +11,17 @@ const Accordion = forwardRef(({ children: originalChildren, allowMultiple, defau
 
   const [expandedItems, setExpandedItems] = useState(defaultIndex);
 
-  const children = useMemo(() => React.Children.toArray(originalChildren), [originalChildren]);
+  const children = useMemo(() => {
```

Comment:
return the type check of the accordion after we found new way to create components with mdx type check